accel/tcg/log_instr.c

```diff
@@ -714,21 +751,55 @@ void qemu_log_instr_global_switch(bool request_stop)
     }
 }
 
+/*
+ * Initialize instruction info entry from the ring buffer.
+ */
+static void qemu_log_instr_info_init(cpu_log_instr_info_t *iinfo)
+{
+    iinfo->txt_buffer = g_string_new(NULL);
```

Comment:
I much prefer C99 bool `true`/`false` instead of the uppercase macros. But glib doesn't...

However, they are compatible (after implicit conversions) and QEMU already passes true/false to many glib functions so maybe use true/false?

Reply:
Sure, I did not look at what the rest of qemu does, will change.